plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcProcedureRelationHandle.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.jdbc;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+
+public class JdbcProcedureRelationHandle
```

Comment:
Make it a record

Reply:
`JdbcProcedureRelationHandle` extends an abstract class. Maybe if we make `JdbcRelationHandle` as an interface then yes. I think it is orthogonal to this PR and it could be a good follow-up. WDYT ?

Reply:
Yep, you're right. I didn't see that. Agree that this is not related